### PR TITLE
catalyst: fix unused git dependency and proper PYTHONPATH

### PR DESCRIPTION
**Pull request by sbna**
fix unused git dependency for version extraction and fix proper PYTHONPATH addition at runtime as paraview

### CODE REVIEW

**Comment by alalazo** [comment]
Hi @sbna, there are a few `flake8` issues with the PR. Fyi, if you want to check them upfront before submitting, you can use:
```console
$ spack flake8
```
The command requires `flake8` to be installed on your workstation.

**Comment by sbna** [comment]
Why is it not still merged?

**Comment by adamjstewart** [comment]
You haven't addressed my last comment yet.

File: var/spack/repos/builtin/packages/catalyst/package.py [hunk 1]
```diff
@@ -128,6 +130,13 @@ def setup_environment(self, spack_env, run_env):
                              paraview_version))
         run_env.prepend_path('LD_LIBRARY_PATH', join_path(lib_dir,
                              paraview_version))
+        if '+python' in self.spec:
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version))
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version, 'site-packages'))
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version, 'site-packages', 'vtk'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/catalyst/package.py]: Do all 3 of these really need to be in `PYTHONPATH`? It's usually just `lib/<whatever>/site-packages` that is necessary.

Comment (sbna) [reply var/spack/repos/builtin/packages/catalyst/package.py]: you are right, it has been fixed in #8752 by @olesenm

**Comment by olesenm** [comment]
Would it make sense to rebase both the paraview and catalyst packages together, squash down some commits and make a single pull request? They are both fairly similar.

**Comment by adamjstewart** [comment]
I'll let both of you decide what to do. A single PR or multiple non-conflicting PRs are both fine by me.

**Comment by sbna** [comment]
I agree with Mark, it's better to have a single PR

**Comment by alalazo** [comment]
> I agree with Mark, it's better to have a single PR

Is there any follow up on this?

**Comment by adamjstewart** [comment]
It looks like part of this PR was handled in #8753, which was superseded by #9641, which was closed.

**Comment by alalazo** [comment]
> It looks like part of this PR was handled in #8753, which was superseded by #9641, which was closed.

:thinking:

**Comment by alalazo** [comment]
@sbna I'm closing this since it has been stale for more than a year and there has been no reply to the latest comments. Feel free to reopen if you disagree or want to continue the PR.

**Comment by sbna** [comment]
sorry for the late reply, you can close it

**Pull request closed without merging**